tests_requre/database/test_models.py

```diff
@@ -494,3 +494,34 @@ def test_get_installation_by_account(
 ):
     assert InstallationModel.get_by_account_login("teg").sender_login == "teg"
     assert InstallationModel.get_by_account_login("Pac23").sender_login == "Pac23"
+
+
+def test_pr_get_copr_builds(clean_before_and_after):
```

Comment:
You may want to use sample values from `SampleClass` and ideally use `fixtures` for the example database objects. (We already define some in `conftest.py`.)

Reply:
Of course, forgot about them, fixed now.